src/components/AutoComplete/AutoCompleteItems.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown, MenuItem } from '../../index';
+
+const AutoCompleteItems = ({
+  items,
+  activeItems,
+  getItemProps,
+  highlightedIndex,
+  selectedItem
+}) => (
+  <Dropdown.Menu
+    style={{
+      display: 'block',
+      position: 'absolute',
+      left: 'inherit',
+      marginTop: 0,
+      top: 'auto'
+    }}
+  >
+    {items.map(({ text, type, disabled = false }, index, items) => {
+      if (type === 'header') {
+        return (
+          <MenuItem key={text} header>
+            {text}
+          </MenuItem>
+        );
+      }
+
+      if (type === 'divider') {
+        return <MenuItem key={`divider-${index}`} divider />;
+      }
+
+      if (disabled) {
+        return (
+          <MenuItem key={text} disabled>
+            {text}
+          </MenuItem>
+        );
+      }
+
+      return (
+        <MenuItem
+          {...getItemProps({
```

Comment:
I think it would be more readable as a separate var
```jsx
const itemProps = getItemProps({ ... });

<MenuItem {...itemProps}>{text}</MenuItem>;
```